app/src/main/java/org/fossasia/openevent/general/auth/AuthService.kt

```diff
@@ -21,6 +22,17 @@ class AuthService(private val authApi: AuthApi,
                 }
     }
 
+    fun signUp(signUp: SignUp): Single<User> {
+        if (signUp.email!!.isEmpty() || signUp.password!!.isEmpty())
```

Comment:
isNotNullOrEmpty

Reply:
@iamareebjamal did you mean isNullorEmpty?

Reply:
Use the method isNullOrEmpty, not signUp.email!!.isEmpty(), this will throw NPE if null